src/gnark_backend_glue/mod.rs

```diff
@@ -197,21 +197,4 @@ mod tests {
         let size = get_exact_circuit_size(&Circuit::default()).unwrap();
         assert_eq!(size, 0);
     }
-
-    //UNUSED TESTS
-    #[test]
-    fn test_verify_should_return_false() {
-        let result = verify_with_meta(Circuit::default(), &[65, 66, 67], &[]).unwrap();
-        assert!(!result);
-    }
-
-    #[test]
-    fn test_prove_should_call_go_backend() {
-        let result = prove_with_meta(Circuit::default(), vec![]).unwrap();
-
-        assert_eq!(
-            std::str::from_utf8(&result).unwrap(),
-            "{\"gates\":[],\"public_inputs\":[],\"values\":[],\"num_variables\":1}"
-        );
-    }
```

Comment:
If these were removed because they're not passing we should use a `#[ignore]` tag on top of the test function name definition explaining why they're being ignored.

Reply:
These were dummy tests I used when writing the Go calls, they are not supposed to work now

Reply:
In that case it's ok to remove them.